server/etcdserver/server.go

```diff
@@ -362,6 +362,15 @@ func NewServer(cfg config.ServerConfig) (srv *EtcdServer, err error) {
 	ci.SetBackend(be)
 	cindex.CreateMetaBucket(be.BatchTx())
 
+	if cfg.ExperimentalBootstrapDefragThresholdMegabytes != 0 {
+		freeableMegabytes := uint(be.Size()-be.SizeInUse()) / 1024 / 1024
+		if freeableMegabytes > cfg.ExperimentalBootstrapDefragThresholdMegabytes {
```

Comment:
Please log here at Info: 
  "Running defragmentation", 
    zap.Field(backend-size-megabytes), 
    zap.Field(backend-in-use-size);  
    zap.Field(experimental-bootstrap-defrag-threshold * 1024 * 1024) 
or 
  "Skipping defragmentation"   ...